components/item/SummaryInfo.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <div class="card rounded-0 border-0 p-4">
     <header
-      v-if="titles"
+      v-if="titles.length > 0"
+      class="mb-3"
```

Comment:
This leaves too large a gap when there is only a title.
Can be seen in the storybook story for 'Only title'

Reply:
OK I've moved the styling to texts & button-links in assets/scss and added a wrapping class .info-panel, the component isn't jumping around anymore. 

The issue with there being too large a gap on pages with only a title was a little hard since the gap is needed and just adding it to the .description doesn't work on pages with no title. But this css works in all cases, only applying the gap if there's a header
```
header + .description {
    margin-top: 1rem;
 }
```